### DISCUSSION TITLE
关于使用非root权限执行安装脚本及DNSPod申请证书疑问

### DISCUSSION THREAD

**Discussion by kekylin** [0xJacky/nginx-ui#328] [Q&A] [answered]
这是一个非常好的项目，感谢大佬的付出。我是一名NAS玩家，在我寻找Nginx Proxy Manager替代品时，发现了这个非常棒的项目，很符合我的使用需求。
当我打算在Ubuntu系统上使用一键脚本安装时，发现不能以sudo权限执行安装脚本，必须使用root权限，请问是否有办法使用sudo权限执行安装？
而在我使用root权限安装之后，在申请泛域名证书时，发现域名托管在DNSPod的无法申请证书，看到备注也显示已弃用。请问以后会考虑支持DSNPod吗？国内很多用户的域名都是托管在腾讯的DNSPod上，无法使用会添加很多的麻烦问题。
感谢付出，
顺遂安康！

**Comment by 0xJacky** [comment, marked as answer]
感谢，

是这样的，DNSPod 国内腾讯的用户，要选择腾讯云（Tencent Cloud DNS）。

非 root 安装现在还暂不支持，如果担心权限问题也可以考虑直接用 docker 镜像

**Reply by kekylin** [reply]
谢谢！明白了。
